metrics/src/main/java/tech/pegasys/pantheon/metrics/MetricCategory.java

```diff
@@ -24,13 +24,14 @@
   PEERS("peers"),
   PROCESS("process", false),
   ROCKSDB("rocksdb"),
+  ROCKSDB_STATS("rocksdb", false),
```

Comment:
Also - might be worth making these category names more accurate while you're in here.  Perhaps: `ROCKSDB_KVSTORE` / `ROCKSDB_KVSTORE_STATS`?

Reply:
I went with `KVSTORE_ROCKSDB` in case we ever switch DBs it has good hierarchy: `KVSTORE_MEMDB`, `KVSTORE_JDBC`, etc.

Reply:
yeah - thats better